site/content@beta/agent-chat-rtm-api/scopes.md

```diff
@@ -0,0 +1,116 @@
+---
+weight: 20
+---
+
+# Scopes
+
+Scopes define the access to certain resources.
+
+- `ro` means read only
+- `rw` means read/write
+- all `rw` scopes contain `ro` privileges
+- all `access` scopes contain `my` privileges
+- all `all` scopes contain `access` privileges and `my` privileges
+
+
+For more details check [here](../services/sso/typedef.go).
+
+## Accessing chat 
+
+This chart shows every possible chats scenarios:
+
+- chats without my access and without my presence (`chats -A -P`)
```

Comment:
co to jest `my presence`? tego nie ma w bullet liście sekcję wyżej

Reply:
Presence oznacza obecność w czacie, tj. ktoś jest uczestnikiem danego czata.